pyth-sdk-terra/README.md

```diff
@@ -38,12 +38,12 @@ This struct also has some useful functions for manipulating and combining prices
 ## Off-Chain Queries
 
 You can use the provided schemas in the `schema` directory to directly query the terra contract from off-chain applications.
-A typical query will look like:
+A typical query requires to pass the price feed id as a hex string. it will look like:
```

Comment:
I removed it from message because terra UI doesn't understand comment and people get confused by copy and paste.